### PR TITLE
Adjusted UI for login

### PR DESCRIPTION
**Pull request by matthewcoufal**
simulator screen shot - iphone 8 plus - 2018-09-14 at 17 31 21

### CODE REVIEW

File: Clicker/ViewControllers/LoginViewController.swift [hunk 1]
```diff
@@ -12,21 +12,51 @@ import SnapKit
...
-        GIDSignIn.sharedInstance().uiDelegate = self
+        gradient = CAGradientLayer()
+        gradient.colors = [UIColor.bluishGreen.cgColor, UIColor.aquaMarine.cgColor]
+        gradient.startPoint = CGPoint(x: 1.0, y: 1.0)
+        gradient.endPoint = CGPoint(x: 0.0, y: 0.0)
+        gradient.frame = view.bounds
+        view.layer.addSublayer(gradient)
+        
+        welcomeLabel = UILabel()
+        welcomeLabel.text = "Welcome to"
```
Comment (kevinchan159) [inline Clicker/ViewControllers/LoginViewController.swift]: can you put the string as a constant class variable (under the var class variables), i.e.:
`let welcomeLabelText = "Welcome to"`

File: Clicker/ViewControllers/LoginViewController.swift [hunk 2]
```diff
@@ -12,21 +12,51 @@ import SnapKit
...
+        welcomeLabel.textColor = .white
+        welcomeLabel.adjustsFontSizeToFitWidth = true
+        welcomeLabel.textAlignment = .center
+        view.addSubview(welcomeLabel)
         
-        appLogoImage = UIImageView(image: #imageLiteral(resourceName: "AppLogo"))
-        appLogoImage.contentMode = .scaleAspectFit
-        view.addSubview(appLogoImage)
+        appNameLabel = UILabel()
+        appNameLabel.text = "Pollo"
```
Comment (kevinchan159) [inline Clicker/ViewControllers/LoginViewController.swift]: same thing as above with "Pollo"

File: Clicker/ViewControllers/LoginViewController.swift [hunk 3]
```diff
@@ -12,21 +12,51 @@ import SnapKit
...
+        appNameLabel.font = ._60HeavyFont
+        appNameLabel.textColor = .white
+        appNameLabel.adjustsFontSizeToFitWidth = true
+        appNameLabel.textAlignment = .center
+        view.addSubview(appNameLabel)
+        
+        pronunciationLabel = UILabel()
+        pronunciationLabel.text = "\"Poh-loh\""
```
Comment (kevinchan159) [inline Clicker/ViewControllers/LoginViewController.swift]: same

File: Clicker/ViewControllers/LoginViewController.swift [hunk 4]
```diff
@@ -35,16 +65,33 @@ class LoginViewController: UIViewController, GIDSignInUIDelegate {
...
     // MARK: - CONSTRAINTS
     func setupConstraints() {
-        appLogoImage.snp.makeConstraints { make in
-            make.center.equalToSuperview()
-            make.width.height.equalTo(70)
+        
+        welcomeLabel.snp.makeConstraints { make in
+            make.width.equalTo(249.5)
+            make.height.equalTo(31.5)
+            make.top.equalToSuperview().offset(194.5)
```
Comment (kevinchan159) [inline Clicker/ViewControllers/LoginViewController.swift]: Make these numbers a constant class variable as well i.e.:
`let welcomeLabelWidth = 249.5`

File: Clicker/ViewControllers/LoginViewController.swift [hunk 5]
```diff
@@ -35,16 +65,33 @@ class LoginViewController: UIViewController, GIDSignInUIDelegate {
...
+            make.height.equalTo(19)
+            make.top.equalTo(appNameLabel.snp.bottom).offset(14.5)
+            make.centerX.equalToSuperview()
         }
         
         signInButton.snp.makeConstraints { make in
-            make.bottom.equalToSuperview().offset(-60)
+            make.width.equalTo(200)
+            make.height.equalTo(40)
+            make.top.equalTo(pronunciationLabel.snp.bottom).offset(28)
```
Comment (kevinchan159) [inline Clicker/ViewControllers/LoginViewController.swift]: same with all these constants

Comment (matthewcoufal) [reply Clicker/ViewControllers/LoginViewController.swift]: Ok, thanks for pointing this out! I'll make those fixes.

**Pull request merged**